src/xrpl-cheat-sheet.json

```diff
@@ -547,6 +547,11 @@
                 "name": "XRPL Buddy",
                 "link": "https://xrpl.buddy.cat",
                 "description": "Public Bot for interacting with XRPL easily from inside Discord"
+            },
+            {
+                "name": "SpaceWatch",
+                "link": "https://spacewatch.app",
+                "description": "The #1 solution for NFT & Token holder verification and role gating in Discord for XRP Ledger"
```

Comment:
Could you describe what the bot does, this list is for people to navigate through xrpl. Would be best to avoid "#1 solution" etc. A simple one line description would be awesome!